cesium/internal/index/domain.go

```diff
@@ -35,7 +35,7 @@ func (i *Domain) Distance(ctx context.Context, tr telem.TimeRange, continuous bo
 	defer func() { err = errors.CombineErrors(err, iter.Close()) }()
 
 	if !iter.SeekFirst(ctx) || (!iter.TimeRange().ContainsRange(tr) && continuous) {
-		err = ErrDiscontinuous
+		err = NewErrDiscontinuousTR(tr)
```

Comment:
May just be a personal preference, but I typically don't put `New` in front of these error constructors. Regardless of which route you choose, please standardize across the cesium codebase

Reply:
IMO it makes more sense to have the actual error stored as `ErrDiscontinuous`, and make a constructor that constructs it. This way we can match the errors easier. How would you do this?

Reply:
Ok thats fine